roles/0-init/tasks/disallow_install_false_with_installed_defined.yml

```
@@ -46,6 +46,12 @@
     fail_msg: "DISALLOWED: 'squid_install: False' (e.g. in /etc/iiab/local_vars.yml) WHEN 'squid_installed' is defined (e.g. in /etc/iiab/iiab_state.yml) -- IIAB DOES NOT SUPPORT UNINSTALLS -- please verify those 2 files especially, and other places variables are defined?"
     quiet: yes
 
+- name: "DISALLOW 'zram_install: False' WHEN zram_enabled is defined"
```

Minor typo, in keeping with this morning's fixup https://github.com/iiab/iiab/commit/8d717c70bc1960d696af583330aedf2477b40ede !

```suggestion
- name: "DISALLOW 'zram_install: False' WHEN zram_installed is defined"
```